pkg/controller/attach_detach.go

```diff
@@ -800,6 +783,45 @@ func (r *ReconcileAttachDetach) recoverAzVolumeAttachment(ctx context.Context, r
 	return nil
 }
 
+func (r *ReconcileAttachDetach) isAttached(ctx context.Context, volumeHandle, nodeName string) bool {
+	w, _ := workflow.GetWorkflowFromContext(ctx)
+	disk, err := r.cloudDiskAttacher.CheckDiskExists(ctx, volumeHandle)
```

Comment:
[`CheckDiskExists`](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/509d944d1b21e66112bc0aaf38047131856f0397/pkg/provisioner/cloudprovisioner.go#L776) will skip the check when in throttling and returns (nil, nil). Should we also check if `disk != nil` to ensure disk's existence?

Reply:
Done.